marshmallow/fields.py

```diff
@@ -616,6 +617,116 @@ def _deserialize(self, value, attr, data, **kwargs):
         return result
 
 
+class Tuple(Field):
+    """A tuple field, composed of a fixed number of other `Field` classes or
+    instances
+
+    Example: ::
+
+        row = Tuple((fields.String(), fields.Integer(), fields.Float()))
+
+    .. note::
```

Comment:
I didn't understand that note until I read the conversation in the PR.

I don't understand if using a Nested field would work out of the box for this use case. If so, they we could at least rephrase as "using a Nested field". Otherwise, if a dedicated field must be written, then I'm not sure what to suggest.

Reply:
I've reworded it to "using a Schema within a Nested field", which reflects better my initial intention of indicating that the user needed to build a schema for the specific named tuple, which is not as obvious if we only mention Nested fields.